src/Symfony/Component/Intl/Data/Bundle/Writer/TextBundleWriter.php

```diff
@@ -193,18 +180,15 @@ private function writeArray($file, array $value, $indentation)
     }
 
     /**
-     * Writes a "table" node.
-     *
      * @param resource $file        The file handle to write to
-     * @param iterable $value       The value of the node
      * @param int      $indentation The number of levels to indent
      * @param bool     $fallback    Whether the table should be merged
      *                              with the fallback locale
      *
      * @throws UnexpectedTypeException when $value is not an array and not a
      *                                 \Traversable instance
      */
-    private function writeTable($file, $value, $indentation, $fallback = true)
+    private function writeTable($file, iterable $value, int $indentation, bool $fallback = true)
     {
         if (!\is_array($value) && !$value instanceof \Traversable) {
```

Comment:
The new type hint makes sure that this condition will always evaluate to `false`. We can remove that `if` block, imho.